domains/mewwme.json

```diff
@@ -0,0 +1,11 @@
+{
+    "description": "mewwme.is-a.dev",
+    "repo": "https://github.com/mewwme/mewwme.github.io",
+    "owner": {
+        "username": "mewwme",
+        "discord": "L`RMN#9749"
+    },
+    "record": {
+        "CNAME": "mewwme.github.io"
```

Comment:
Content loop. No content is shown. Redirects back to the domain you are requesting.